CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ list(APPEND THIRD_PARTY_MODULES timelib)
 list(APPEND EXTRA_INCLUDE_PATHS "${TP_DIR}/timelib")
 
 list(APPEND THIRD_PARTY_MODULES folly)
+list(APPEND EXTRA_INCLUDE_PATHS "${TP_DIR}/folly")
```

Comment:
This is relying on the symlink. You can't rely on the symlink. This needs to be `folly/src` instead.